src/support/SupportResourceList.tsx

```diff
@@ -0,0 +1,85 @@
+import _ from 'lodash/fp';
+import React from 'react';
+import * as Style from 'src/libs/style';
+import * as Utils from 'src/libs/utils';
+import { LookupSummaryAndPolicies } from 'src/support/LookupSummaryAndPolicies';
+import { SupportResourceListItem, SupportResourceListItemProps } from 'src/support/SupportResourceListItem';
+import { supportResources, SupportResourceType } from 'src/support/SupportResourceType';
+
+interface SupportResourceListProps {
+  queryParams: {
+    resourceType: string | undefined;
+    resourceId: string | undefined;
+  };
+}
+
+export const SupportResourceList = (props: SupportResourceListProps) => {
+  const selectedType = props.queryParams.resourceType || '';
+  const resourceId = props.queryParams.resourceId || '';
+  const supportResourceListWidth = 350;
+
+  const makeResourceListItemProps = (resourceType: SupportResourceType): SupportResourceListItemProps => {
+    return {
+      resourceType,
+      isActive: !!selectedType && resourceType.resourceType === selectedType,
+    };
+  };
+
+  return (
+    <div role='main' style={{ display: 'flex', flex: 1, height: `calc(100% - ${Style.topBarHeight}px)` }}>
```

Comment:
Again, this shouldn't have role main because the containing page already uses that role.